package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sentry-cli",
+  "version": "0.25.0",
+  "description": "A command line utility to work with Sentry. https://docs.sentry.io/hosted/learn/cli/",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "preinstall": "INSTALL_DIR=./bin ./bin/install-sentry-cli"
```

Comment:
Is there a way that we can just leverage our existing install script without duplicating it in two places?

If there are changes to name to the script to accommodate working for both, I think we'd be more inclined to support that, in one place, than a copy/paste in two places where one that we never use will absolutely break at some point in the future. :)